cmake/projects/glfw/hunter.cmake

```diff
@@ -16,12 +16,23 @@ hunter_add_version(
     PACKAGE_NAME
     glfw
     VERSION
+    "3.2.1"
+    URL
+    "https://github.com/hunter-packages/glfw/archive/3.2.1.gz"
```

Comment:
There is no such release.

Reply:
My bad, wrong repo: https://github.com/glfw/glfw/archive/3.2.1.tar.gz

Reply:
> My bad, wrong repo: https://github.com/glfw/glfw

Will not work, this package need "hunterization":
* https://docs.hunter.sh/en/latest/creating-new/create/cmake-dependencies.html

That's why fork is used.